roles/grafana/templates/prometheus_source.yml.j2

```diff
@@ -0,0 +1,23 @@
+
+datasources:
+- name: Prometheus
+  type: prometheus
+  access: proxy
+  url: http://localhost:9090
```

Comment:
Should this URL be parametrized in case Prometheus is not installed on the same host as Grafana?

Reply:
Added Promethes_url parameter